Dockerfile

```diff
@@ -4,11 +4,13 @@ LABEL description="Sentry Issues & Events Exporter"
 
 WORKDIR /app
 
+COPY requirements.txt .
```

Comment:
requirements.txt changes way less than actual code. Putting it before the main code copy allows to leverage docker build cache for steps that rarely change. Image build will be faster and take less space.